### PR TITLE
Use Pandoc div syntax for (aspirational) tips about RStudio and workf…

### PR DESCRIPTION
**Pull request by jennybc**
…low/lifestyle

tagging you in @hadley

### CODE REVIEW

**Comment by hadley** [comment]
Note that I don't see the images in the RStudio preview, but I do when I pop the chapter out in to the browser.

File: style.css [hunk 1]
```diff
@@ -12,3 +12,45 @@ pre {
...
+  background-position: 5px center;
+}
+
+div.rstudio-tip {
+  background-image: url("../diagrams/lightbulb.png");
+}
+
+div.tip {
+  background-image: url("../rstudio.png");
+
```
Comment (jennybc) [inline style.css]: I assume this empty line can go?

File: style.css [hunk 2]
```diff
@@ -12,3 +12,45 @@ pre {
...
+}
+
+div.base::before {
+  display: block;
+  content: "In base R";
+}
```
Comment (jennybc) [inline style.css]: Do you think we'll use these too or should we delete this?

**Comment by hadley** [comment]
I think this is good to go — we can certainly tweak the images and add more options in the future.

**Pull request merged**